README/ReadMe.txt

```diff
@@ -0,0 +1,3 @@
+How to start project
+1.Write to command line update-database -context DbContextSellCar
```

Comment:
![image](https://user-images.githubusercontent.com/3023032/220547008-f182628c-e9bb-42ba-8c30-0e40872ac9ac.png)
does not work